mantle/kola/harness.go

```diff
@@ -642,41 +659,23 @@ func runProvidedTests(tests map[string]*register.Test, patterns []string, multip
 	firstRunErr := suite.Run()
 	firstRunErr = handleSuiteErrors(outputDir, firstRunErr)
 
-	if len(failedTests) > 0 && rerun {
+	var rerunPatterns []string
```

Comment:
```suggestion
	var rerunTests []string
```

`rerunPatterns` implies that the values aren't absolute names (i.e. they could be a regex or something), but that's not the case here is it?

Reply:
It might also be nice to put a comment above this block.. Something that explains why we need to do some translation from `h.Name()` to another string for our `rerunTests` and also why `h.Name() == "non-exclusive-tests"` is special.